src/simulation.py

```diff
@@ -76,19 +78,30 @@ def new_gen(self):
                                reproduction(self.config, self.selection(self.population.organisms), self.selection(self.population.organisms)))
             new_population.organisms[organism] = new_org
         return new_population
+    
+    def stats(self):
+        avg_score = 0
+        survival_rate = 0
+        for organism in self.population.organisms:
+            avg_score += organism.score / self.config.population_size
+            if organism.alive:
+                survival_rate += (1 / self.config.population_size) * 100
+        return round(avg_score, 2), round(survival_rate, 2)
 
 
     def reset(self):
+        stats = self.stats()
+        print("Average Score:", stats[0],f"\nSurvival Rate: {stats[1]}%")
```

Comment:
Memo: maybe refactor that into something displayed on screen / a "nice" debug message